digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/util/ProjectLinkSplitter.scala

```diff
@@ -101,10 +104,11 @@ object ProjectLinkSplitter {
       Seq(splits._1, splits._2, splits._3).filter(pl => Math.abs(pl.endMValue - pl.startMValue) >= fi.liikennevirasto.viite.MinAllowedRoadAddressLength)
     }
     val suravageM = GeometryUtils.calculateLinearReferenceFromPoint(split.splitPoint, suravage.geometry)
+    val suravageSplitHeading = GeometryUtils.calculateHeadingFromLinearReference(suravage.geometry, suravageM)
     val templateM = GeometryUtils.calculateLinearReferenceFromPoint(split.splitPoint, templateLink.geometry)
     val splitAddressM = templateLink.startAddrMValue + Math.round(templateM / templateLink.geometryLength *
       (templateLink.endAddrMValue - templateLink.startAddrMValue))
-    val isReversed = (split.statusB == LinkStatus.New) ^ isDirectionReversed(suravage, templateLink)// isTailConnected(suravage, templateLink)
+    val isReversed = (split.statusB == LinkStatus.New) ^ isDirectionReversed(suravage, templateLink)
```

Comment:
Check that split A and split B are properly adjusted according to the new specs and won't break this reversion check (B is always LinkStatus.New now)